### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Sharma-palak**
@bvishal8510  sir plzz review the pr

### CODE REVIEW

**Comment by Sharma-palak** [comment]
@bvishal8510  sir plzz review

File: core/templates/developer.html [hunk 1]
```diff
@@ -18,12 +18,29 @@
     <div class="container-fluid">
         {% include 'header.html' %}
         {{ display_name|safe }}
+         {{content|safe|escape}}
```
Comment (bvishal8510) [inline core/templates/developer.html]: add page.content here because you are not sending anything named content

File: core/templates/event.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                {{ display_name|safe }}
+
+ <div class="col-lg-10 col-md-10 col-sm-12 mx-auto center_content" style="min-height:375px">
+{% if event %}
+{% for events in event %}
+
+<div class="col-lg-10 col-md-10 col-sm-12 mx-auto shadow" style="margin:50px 0px 50px 0px;">
+            <div class="row" style="padding:20px 0 20px 0;font-weight: 700">
+                <div class="col-12" > <h3 class="evnname"style="font-size:35px;font-weight:Bold;letter-spacing:2px;">{{events.name}}</h3></div>
+                <div class="col-12"  style="padding:px ;min-height:400px; width: inherit " >{{events.description}}</div>
```
Comment (bvishal8510) [inline core/templates/event.html]: use |safe in the event description to get only content not the tags.

File: core/templates/index.html [hunk 3]
```diff
@@ -126,49 +127,98 @@ <h5>Student Feedback</h5>
...
-                <p><i class="fas fa-feather-alt" style="font-size:24px; margin-right:10px;"></i>Education is the key determinant of how the student utilizes the skills imparted for self-centered purposes or also for broader societal & environmental good. Human education facilitates human conduct and the development of humane society, otherwise education results in inhuman conduct and degradation of society and environment. We are witness to the conduct exhibited by the students, not only here, but world-wide. Due to lack of right understanding we have reached a point where we can clearly see the results of our collective education system problems such as pollution, resource depletion, and extinction of animals, global warming, terrorism and even threats to the human race on earth.
-                    
-                    This need for value based education has been repeatedly highlighted by various commissions and expert committees on education (Dr. Radhakrishnan, Dr. Kothari, UNESCO declaration on service to society and so on).
-                    
+            <div class="col-lg-6 col-sm-12 my-2 about-text mx-auto ">
```
Comment (bvishal8510) [inline core/templates/index.html]: reduce about-us content a bit more just enough to level with images

File: core/urls.py [hunk 4]
```diff
@@ -27,5 +29,14 @@
     url(r'^savecontact$', SaveContactView.as_view(), name='save_contact'),
     url(r'^$', IndexView.as_view(), name='home'),
     url(r'^developer/$',Developer_View.as_view() ,name ='developer'),
+    url(r'^club_about/$', Nature_Club_About.as_view(), name='club_about'),
+    url(r'^club_activity/$', Nature_Club_Activity.as_view(), name='club_activity'),
+    url(r'^health_about/$', Health_Club_About.as_view(), name='health_about'),
+    url(r'^health_activity/$', Health_Club_Activity.as_view(), name='health_activity'),
+    url(r'^event/(?P<id>[0-9]+)/$', Event_Page.as_view(), name='event_page'),
+    # url(r'^notification/(?P<id>[0-9]+)/$', Notification_View.as_view(), name='notification'),
```
Comment (bvishal8510) [inline core/urls.py]: remove this

File: core/views.py [hunk 5]
```diff
@@ -311,6 +313,105 @@ def get(self, request, *args, **kwargs):
...
+            event2 = Event.objects.filter(id=id)
+            dict.extend(event2)
+            print(dict)
+            event1 = Event.objects.exclude(id=id).order_by("-id")
+            print(event1)
+            dict.extend(event1)
+            print(dict)
+            return render(request, self.template_name, context={'event':dict,'display_name': display_name,})
+
+
```
Comment (bvishal8510) [inline core/views.py]: why all these blank lines??

**Comment by Sharma-palak** [comment]
@bvishal8510  sir done with the changes

File: core/templates/index.html (hunk 3 above)
Comment (Sharma-palak) [reply core/templates/index.html]: @bvishal8510  sir mere me images ke level pr dikha raha hai aur agr aur reduce karti hu to recent events aur notification upar neeche ho jaayenge

**Comment by Sharma-palak** [comment]
@bvishal8510  sir done with rest of changes

File: core/views.py [hunk 6]
```diff
@@ -311,6 +313,105 @@ def get(self, request, *args, **kwargs):
...
+    def get(self,request,id=None,*args,**kwargs):
+        display_name = "<div class='col-lg-12 mx-auto '><h3 class=' my-2'><span class='about-us'>Events</span> </h3></div>"
+        if(id==None):
+            event = Event.objects.all().order_by("-id")
+            return render(request,self.template_name, context={'event':event,'display_name': display_name,})
+        else:
+            dict=[]
+            event2 = Event.objects.filter(id=id)
+            dict.extend(event2)
+            print(dict)
```
Comment (bvishal8510) [inline core/views.py]: remove print statements

**Comment by Sharma-palak** [comment]
@bvishal8510 sir removed print statement

**Comment by Sharma-palak** [comment]
@bvishal8510  sir ,i have squashed commits

**Pull request merged**